### PR TITLE
note pillow dependency

### PR DESCRIPTION
**Pull request by M3ssman**
Added small addition for Python Dependency and explanation when installing tesstrain. Otherwise running into 

```
Traceback (most recent call last):
  File "generate_line_box.py", line 6, in <module>
    from PIL import Image
ModuleNotFoundError: No module named 'PIL'

```
issue when starting to train

### CODE REVIEW

**Comment by wrznr** [comment]
I am hesitant to add a `requirements.txt`.
1. This is not a Python project. If we want to make it one, a few other things have to be changed.
2. We do not provide build files or dependency checks for `make` or `tesseract`. Yet, both are necessary to run the training.

**Comment by M3ssman** [comment]
I agree, 
this is not a common Python project and far away from the consistent style of other OCR-D module projects, although it already includes software-management artifacts like a `.pylintrc` at root level (which, in turn, might be a legacy thing).

On the other hand, the targets provided to build both leptonica and tesseract are shaped in usual Makefile-manner, which includes sourcecode-downloads and dependency checks as well and will fail rather verbose if something can't be done as expected - assuming `wget`and regular build-tools are on board.

IMHO, adding a clear marker for Python dependencies increases project transparency and eases the pain to get started. In fact, without Python, i.e. `python3`, and without Pillow, it is impossible to do anything. This functional dependencies must be communicated in a conventional way, which is, in the Python-Universe, a Dependency-File.

**Comment by wrznr** [comment]
@M3ssman Okay. You convinced me: Let's have `requirements.txt`. With respect to your modification of the README, why is having `pip` a must?

File: README.md [hunk 1]
```diff
@@ -24,6 +24,10 @@ autotools (including autotools-archive) and some additional libraries for the
 training tools. See the [installation notes in the tesseract
 repository](https://github.com/tesseract-ocr/tesseract/blob/master/INSTALL.GIT.md).
 
+### python
```
Comment (stweil) [inline README.md]:
```suggestion
### Python
```

File: README.md [hunk 2]
```diff
@@ -24,6 +24,10 @@ autotools (including autotools-archive) and some additional libraries for the
...
+
+You need a recent version of Python 3.x and `pip`. For Image-Processing the Python-Library `Pillow` is used. If you don't have a global Installation, please use the provided Requirements-file `pip install -r requirements.txt`.
```
Comment (stweil) [inline README.md]:
```suggestion
You need a recent version of Python 3.x. For image processing the Python library `Pillow` is used.
If you don't have a global installation, please use the provided requirements file `pip install -r requirements.txt`.
```

**Comment by M3ssman** [comment]
@wrznr Using pip is the common way to install Python packages. To be honest, I personally never used a different tool to manage Python Dependencies. How do you handle this? Do you prefer to clone projects repositories .. ? Well, I guess `pip`is in fact part of a *nix-Python-Installation. The `python-pip`version bundled with official Ubuntu 18.04 LTS Server is outdated (9.x-something instead of current 19.x) and has Issues resolving Dependencies that require wheels, so it is necessary to upgrade `pip` as well _before_ it is really doing what it's supposed to do.

@stweil it's a good point to mention the global installation. When working on shared remote Hosts, with automated CI/CD-Services, it is recommended to use separate `venv` for each project. Therefore, I didn't look that direction.

**Comment by stweil** [comment]
Installing `python-pip` which provides `pip` would be wrong because that is for Python 2.x. Installing `python-pip3` is not needed when using virtual environments because venv typically provides `pip` and `pip3`. That's why I'd not say "You need [...] pip".

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+Pillow==6.2.1
```
Comment (kba) [inline requirements.txt]: `>=` would suffice

**Comment by wrznr** [comment]
@kba Me neither. That's strange.

**Comment by wrznr** [comment]
@M3ssman Could you please modify your PR as @stweil suggests?

**Comment by stweil** [comment]
@wrznr, could you please try once more? I just modified the repository settings. You already were an admin, but maybe that rights were restricted by the team settings.

**Comment by wrznr** [comment]
@stweil Hah: It is because I can not write to https://github.com/ulb-sachsen-anhalt/tesstrain what is perfectly fine. We will have to wait for @M3ssman .

**Comment by kba** [comment]
Can u please make it a >= requirement? Otherwise there might be version issues

**Comment by kba** [comment]
LGTM

**Comment by M3ssman** [comment]
@all Confused me: PR still hangs at `Changes requested`?

**Comment by kba** [comment]
Yeah, but it is ready for merge. @stweil @wrznr feel free I'm afk

**Comment by wrznr** [comment]
I rerequested a review by @stweil. Let's wait until it is done.

**Comment by stweil** [comment]
Thank you, @M3ssman (and also to the reviewers).

**Pull request merged**